build/rpmbuild-docker/Dockerfile

```diff
@@ -25,11 +25,13 @@ ENV GO_VERSION 1.13.7
 ENV GO_CHECKSUM b3dd4bd781a0271b33168e627f7f43886b4c5d1c794a4015abf34e99c6526ca3
 
 RUN wget --progress=dot:giga https://dl.google.com/go/go${GO_VERSION}.linux-amd64.tar.gz -O /tmp/golang.tar.gz && \
-    echo "${GO_CHECKSUM} /tmp/golang.tar.gz" | sha256sum -c -;
-RUN tar -C /usr/local -xzf /tmp/golang.tar.gz; \
+    echo "${GO_CHECKSUM} /tmp/golang.tar.gz" | sha256sum -c - && \
+    tar -C /usr/local -xzf /tmp/golang.tar.gz; \
```

Comment:
We use `--squash` do build this Docker image; there is no need to use `&&` pattern.
We should also not mix tabs and spaces.
We should also not mix things that should merged (that change) and things that should not (submodule branch update for FB) in one PR.